src/pages/MissionAndPurpose/MissionAndPurpose.js

```diff
@@ -13,21 +13,23 @@
 import React from 'react';
 import Container from 'components/Container/Container.js';
 import Layout from 'components/Layout/Layout.js';
+import TitleBody from 'components/TitleBody/TitleBody.js';
 
 const MissionAndPurpose = ({ data }) => {
   if (!data) return null;
 
   let { builder } = data;
-
+  
   return(
     <Layout>
       <Container>
         <div className="about-mission-purpose">
```

Comment:
Do you need this classname any more?

Reply:
I put it in there to target h4s in the dangerouslySetInnerHTML.

See next comment